content/en/Integrations/webhooks.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Get Pentest Updates with Webhooks"
+linkTitle: "Webhooks"
+weight: 10
+description: >
+  Set up webhook notifications for your organization.
+---
+
+{{% pageinfo %}}
+Configure webhooks to subscribe to real-time updates for your pentests.
+{{% /pageinfo %}}
+
+With our API-based webhooks, you can set up an integration between your app and the Cobalt platform to get notifications for pentest events. We’ll send you updates for each event to your URL through an HTTP POST request.
+
+When you work with an API, you can become aware of new data in the following ways:
+
+- Repeatedly send requests to the same [API endpoint](/getting-started/glossary/#api-endpoint) to retrieve new information. This process is known as API polling.
+- Configure a webhook that automatically sends updates for new data to the specified URL.
```

Comment:
Based on my previous comment, I might:

```suggestion
When you work with our webhooks, you can monitor progress with your pentests to:

- Use polling to retrieve new information on a regular basis.
- Configure a webhook that automatically sends updates to the specified URL.
```

I don't think we need to specify "API polling", as it's functionally no different from "regular" [polling](https://en.wikipedia.org/wiki/Polling_(computer_science))

I think "updates" and "for new data" are somewhat redundant.

Reply:
Alternative:

```suggestion
When you work with an API, you can become aware of new data in the following ways:

- Repeatedly send requests to the same [API endpoint](/getting-started/glossary/#api-endpoint) to retrieve new information.
- Configure a webhook that automatically sends updates for new data to the specified URL.
```